documentation/disaster-recovery.md

```diff
@@ -2,63 +2,99 @@
+
+If this document is being followed as part of a DR test, then the following steps should have already been completed by the devops team
+- [create DR test documentation requirements](https://github.com/DFE-Digital/teacher-services-cloud/blob/main/documentation/disaster-recovery-testing.md#documentation-requirements)
+- either [DR test Scenario 1 setup](https://github.com/DFE-Digital/teacher-services-cloud/blob/main/documentation/disaster-recovery-testing.md#scenario-1-loss-of-database-instance) or [DR test Scenario 2 setup](https://github.com/DFE-Digital/teacher-services-cloud/blob/main/documentation/disaster-recovery-testing.md#scenario-2-loss-of-data) depending on the scenario being tested
 
 ## Scenario 1: [Loss of database server](https://technical-guidance.education.gov.uk/infrastructure/disaster-recovery/#loss-of-database-instance)
 
-In case the database server is lost, the objectives are:
+In this scenario, the [Azure Postgres flexible server](https://portal.azure.com/?feature.msaljs=true#browse/Microsoft.DBforPostgreSQL%2FflexibleServers) and the database it contains have been completely lost.
 
-- Recreate the lost postgres database server
-- Restore data from nightly backup stored in Azure.
+As the point-in-time and snapshot backups created by the Azure Postgres service may not be available if it's been deleted or if there is an Azure region issue, the Postgres server will have to be recreated and the database restored from the nightly github workflow scheduled database backups. These backups are stored in [Azure storage accounts](https://portal.azure.com/?feature.msaljs=true#browse/Microsoft.Storage%2FStorageAccounts) and kept for 7 days.
 
-The point-in-time and snapshot backups created by the Azure Postgres service may not be available if it's been deleted, or if there is an Azure region issue.
+The objectives are:
 
-### Start the incident process
+- Recreate the lost postgres database server
+- Restore data from nightly backup stored in Azure
+
+### Start the incident process (if not already in progress)
 Follow the [incident playbook](https://tech-docs.teacherservices.cloud/operating-a-service/incident-playbook.html) and contact the relevant stakeholders as described in [create-an-incident-slack-channel-and-inform-the-stakeholders-comms-lead](https://tech-docs.teacherservices.cloud/operating-a-service/incident-playbook.html#4-create-an-incident-slack-channel-and-inform-the-stakeholders-comms-lead).
 
+### Freeze pipeline
+
+Alert developers that no one should merge to main.
+- In github setings, a user with repo admin privileges should update the *Branch protection rules* and set required PR approvers to 6
+
 ### Enable maintenance mode
 
 Run the *enable maintenance workflow* for the service and environment affected.
 
+If the maintenance page message needs to be updated at any time during the incident
```

Comment:
Can this be moved to https://github.com/DFE-Digital/teacher-services-cloud/blob/main/documentation/maintenance-page.md#update-content ?

Reply:
moved and added link